documentation/docs/03-loading.md

```diff
@@ -78,7 +78,7 @@ SvelteKit's `load` receives an implemention of `fetch`, which has the following
 
 > Code called inside `load` blocks:
 >
-> - should use the SvelteKit-provided [`fetch`](#loading-input-fetch) wrapper rather than using the native `fetch`
+> - should use the SvelteKit-provided [`fetch`](#loading-input-fetch) wrapper rather than using the native `fetch`. Although, using native `fetch` will work, it will cause the API server to be called twice - during SSR and hydration
```

Comment:
How about

"... using the native `fetch` as the latter will make the request twice each during SSR and hydration"

Reply:
yes, this sound better